components/base/wheeled/kinematics.go

```diff
@@ -23,13 +20,21 @@ type kinematicWheeledBase struct {
 
 // WrapWithKinematics takes a wheeledBase component and adds a slam service to it
 // It also adds kinematic model so that it can be controlled.
-func WrapWithKinematics(ctx context.Context, base *wheeledBase, slam slam.Service) (base.KinematicBase, error) {
+func (base *wheeledBase) WrapWithKinematics(ctx context.Context, slamSvc slam.Service) (base.KinematicBase, error) {
 	var err error
+	wb, ok := utils.UnwrapProxy(base).(*wheeledBase)
```

Comment:
[minor] Does
```suggestion
	wb, ok := base.(*wheeledBase)
```
work? in preparation for UnWrap proxy being removed, since we're in the same package.

Reply:
This does not work as the type is a `reconfigurableBase`

Reply:
Wait, is it? reconfigurables are in the `base.go` package. Actually, isn't this passed in from line 23?

Reply:
Actually good catch.  This was leftover code from before